### DISCUSSION TITLE
Listed Databases (Tmax Tibero)

### DISCUSSION THREAD

**Discussion by cklee90** [apache/superset#25946] [Q&A / Help]
I have reviewed the documentation of Apache Superset, and have some questions regarding installing database drivers.

https://superset.apache.org/docs/databases/installing-database-drivers/

According to the link above, I have checked the list of supported databases, and there is no Tibero on the list, which I want to use.

1) Is it possible to connect 'any' database using JDBC or ODBC protocol?
2) If it is not possible to connect, what are the required steps that I have to proceed in order to use Apache Superset on Tmax Tibero 6 and 7?

Thank you for your time.

**Comment by applewyw** [comment]
Have you solved it?

**Comment by dadaiksunn** [comment]
https://pypi.org/project/django-tibero/
pip install django-tibero
